play-services-core/src/main/kotlin/org/microg/gms/auth/signin/AuthSignInActivity.kt

```diff
@@ -197,18 +200,37 @@ class AuthSignInActivity : AppCompatActivity() {
 
     private fun finishResult(statusCode: Int, message: String? = null, account: Account? = null, googleSignInAccount: GoogleSignInAccount? = null) {
         val data = Intent()
-        if (statusCode != CommonStatusCodes.SUCCESS) data.putExtra("errorCode", statusCode)
-        data.putExtra("googleSignInStatus", Status(statusCode, message))
-        data.putExtra("googleSignInAccount", googleSignInAccount)
+        if (statusCode != CommonStatusCodes.SUCCESS) data.putExtra(AuthConstants.ERROR_CODE, statusCode)
+        data.putExtra(AuthConstants.GOOGLE_SIGN_IN_STATUS, Status(statusCode, message))
+        data.putExtra(AuthConstants.GOOGLE_SIGN_IN_ACCOUNT, googleSignInAccount)
+        val bundle = Bundle()
         if (googleSignInAccount != null) {
-            data.putExtra("signInAccount", SignInAccount().apply {
+            SignInAccount().apply {
                 email = googleSignInAccount.email ?: account?.name
                 this.googleSignInAccount = googleSignInAccount
-                userId = googleSignInAccount.id ?: getSystemService<AccountManager>()?.getUserData(account, "GoogleUserId")
-            })
+                userId = googleSignInAccount.id ?: getSystemService<AccountManager>()?.getUserData(
+                    account,
+                    AuthConstants.GOOGLE_USER_ID
+                )
+            }.let {
```

Comment:
Why did you change this to a `let` expression?

Reply:
it changed